src/BibleWell.Aquifer.Data/Constants.cs

```diff
@@ -0,0 +1,20 @@
+namespace BibleWell.Aquifer.Data;
+
+public static class Constants
+{
+    private const string DatabaseFilename = "BibleWellAquifer.db3";
+
+    // Flags are not used with Microsoft.Data.Sqlite.Core 
+    // public const SQLite.SQLiteOpenFlags Flags =
+    //     SQLite.SQLiteOpenFlags.ReadWrite |
+    //     SQLite.SQLiteOpenFlags.Create |
+    //     SQLite.SQLiteOpenFlags.SharedCache;
+
+    // Might have to add an application specific dir to this path
+    public static string DatabasePath =>
+        Path.Combine(
+            Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData),
```

Comment:
I think we'll want to get the correct storage location from a platform specific call (mobile should use MAUI Essentials).  This works for now but we should do that update in a future PR after I have the platform implementations and interfaces worked out.